process_manager/scheduler.py

```diff
@@ -0,0 +1,8 @@
+import time
+
+
+class Scheduler():
+
+    def new_schedule(self, name):
+        time.sleep(5)
```

Comment:
In order to test properly, instead of a fixed sleep, we should probably make this a random time.